Sources/SemanticIndex/UpdateIndexStoreTaskDescription.swift

```diff
@@ -430,11 +430,6 @@ private func adjustSwiftCompilerArgumentsForIndexStoreUpdate(
     .option("output-file-map", [.singleDash], [.separatedBySpace, .separatedByEqualSign]),
   ]
 
-  let removeFrontendFlags = [
-    "-experimental-skip-non-inlinable-function-bodies",
-    "-experimental-skip-all-function-bodies",
-  ]
-
```

Comment:
Don't we want to remove these for indexing?

Reply:
They would presumably only be there if we added them, which we're currently. not. Probably doesn't hurt to keep anyway though.